simapp/app.go

```diff
@@ -397,7 +397,6 @@ func NewSimApp(
 	app.ModuleManager.RegisterInvariants(&app.CrisisKeeper)
 	app.ModuleManager.RegisterRoutes(app.Router(), app.QueryRouter(), encodingConfig.Amino)
 	app.configurator = module.NewConfigurator(app.appCodec, app.MsgServiceRouter(), app.GRPCQueryRouter())
-	app.ModuleManager.RegisterServices(app.configurator)
```

Comment:
Already called once in [runtime/app](https://github.com/cosmos/cosmos-sdk/blob/90759acc90587e00e2bb4bc64c70909f43b5562e/runtime/app.go#L75).